src/seedu/addressbook/ui/TextUi.java

```diff
@@ -7,7 +7,6 @@
 
 import java.io.InputStream;
 import java.io.PrintStream;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 import java.util.Scanner;
```

Comment:
How about the DIVIDER variable? There are some components still in UI that can be shifted to the formatter class.